packages/shared/lib/auxiliary/deep-link/handlers/governance/operations/handleDeepLinkAddProposalOperation.ts

```diff
@@ -1,9 +1,12 @@
 import { showAppNotification } from '@auxiliary/notification/actions'
 import { closePopup, openPopup } from '@auxiliary/popup/actions'
 import { isValidUrl } from '@core/utils/validation'
-import { isProposalAlreadyAdded, isValidProposalId } from '@contexts/governance/utils'
+import { getProposalFromEventId, isProposalAlreadyAdded, isValidProposalId } from '@contexts/governance/utils'
 
 import { AddProposalOperationParameter } from '../../../enums'
+import { selectedProposal } from '@contexts/governance'
```

Comment:
Should be `@contexts/governance/stores`

This isn't technically in the coding conventions, and David and I are working on a way to not have to do that we'll add to the conventions.